src/index.js

```diff
@@ -5,37 +5,68 @@
 import './css/base.scss';
 import fetcher from './fetch.js';
 import User from './User.js';
+import domUpdates from './DOM-updates.js';
+
+let selectionBox = document.querySelector('.selection-box');
```

Comment:
Sidenote, `selectionBox` sounds like a Spartan type room where we choose if children are strong enough to live in our society

Reply:
Brb, redoing this project to make it about selecting strong Spartan children